testsuite/python/lb_stokes_sphere.py

```diff
@@ -32,39 +32,43 @@
 import numpy as np
 import sys
 
-
-@ut.skipIf(not has_features(["LB_GPU", "LB_BOUNDARIES_GPU"]),
-           "Features not available, skipping test!")
-class Stokes(ut.TestCase):
+# Define the LB Parameters
+TIME_STEP = 0.4
+AGRID = 1.0
+KVISC = 5.0
+DENS = 1.0
+LB_PARAMS = {'agrid': AGRID,
+             'dens': DENS,
+             'visc': KVISC,
+             'fric': 1.0,
+             'tau': TIME_STEP}
+# System setup
+radius = 5.4
+box_width = 44
+real_width = box_width + 2 * AGRID
+box_length = 50
+v = [0, 0, 0.01]  # The boundary slip
+
+
+class Stokes(object):
+    lbf = None
+    system = espressomd.System(box_l=[real_width, real_width, box_length])
+    system.box_l = [real_width, real_width, box_length]
+    system.time_step = TIME_STEP
+    system.cell_system.skin = 0.4
+
+    # The temperature is zero.
+    system.thermostat.set_lb(kT=0)
```

Comment:
this has to be done in the test method because the guard for the feature does not catch this statement which is not available when neither ```LB``` nor ```LB_GPU``` is compiled in.

Reply:
are you sure about that? I see that the thermostat should be set in the test method but skin is set within the main class in other test cases also (lb.py, lb_poiseuille.py)!

Reply:
yes I only meant the thermostat